oaknational/Oak-Web-Application
# Feat/feature flag

## Description

- adds feature flag functionality
- fixes #885 behind a feature flag called 'test-feature-flag' 
- see https://app.posthog.com/feature_flags

## Issue(s)

Fixes #885

## How to test

1. Go to blog page
2. Click on copy link button
3. You should see nice animation and only normal focus state


## Checklist
